dev/ProgressBar/ProgressBar.xaml

```diff
@@ -11,11 +11,11 @@
         <Setter Property="Background" Value="{ThemeResource ProgressBarBackground}" />
         <Setter Property="BorderThickness" Value="{ThemeResource ProgressBarBorderThemeThickness}" />
         <Setter Property="BorderBrush" Value="{ThemeResource ProgressBarBorderBrush}" />
-        <Setter Property="MinHeight" Value="{ThemeResource ProgressBarThemeMinHeight}" />
+        <Setter Property="MinHeight" Value="{StaticResource ProgressBarMinHeight}" />
         <Setter Property="Maximum" Value="100" />
         <Setter Property="IsTabStop" Value="False" />
         <Setter Property="VerticalAlignment" Value="Center" />
-        <contract7Present:Setter Property="CornerRadius" Value="{ThemeResource ControlCornerRadius}" />
+        <contract7Present:Setter Property="CornerRadius" Value="{StaticResource ProgressBarCornerRadius}" />
```

Comment:
Do we deliberately want to use a custom corner radius instead of the common one shared by all controls? (Not saying that's wrong, just double-checking.)

Reply:
yes, needs to be rounded. 4px for controls vs 1.5 we want here.